.claude/agents/pull-requests.md

```
@@ -0,0 +1,645 @@
+
+| Rama | Propósito | Protegida | Merge permitido desde |
+|---|---|---|---|
+| `main` | Producción — código desplegado en PRO | ✅ Sí | `develop` (solo release tags) |
+| `develop` | Integración continua — rama base de todas las features | ✅ Sí | PRs de feature/fix/chore/test |
+
+> ⚠️ **Nunca** crear ramas directamente desde `main`.  
+> ⚠️ **Solo el Orchestrator** crea ramas. Los agentes especializados trabajan sobre la rama asignada.
+
+### Ramas de trabajo (corta vida)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)